### ISSUE TITLE
Keystone support for Asus Strix Scar G733 (G533, g732, g532 etc)

### ISSUE THREAD

**Issue by Valieno** [seerge/g-helper#1996]
### Rules

- [X] I made myself familiar with the <a href='https://github.com/seerge/g-helper?tab=readme-ov-file'>Readme</a>, <a href='https://github.com/seerge/g-helper/wiki/FAQ'>FAQ</a> and <a href='https://github.com/seerge/g-helper/wiki/Troubleshooting'>Troubleshooting</a>.
- [X] I understand that, if insufficient information will be provided, my issue will be closed without an answer.

### Is your feature request related to a problem? Please describe

It would be nice to have some support for Keystone functionality

### Describe the solution you'd like

When Keystone inserted/removed launch an app or mode/profile change

### Describe alternatives you've considered

Basically Keystone is an NFC key and i wonder if its possible to make it as Log In NFC key for Windows Hello feature

### Device and Model

G733, G533, G532 etc

### Additional information.

Armoury Crate has some basic functions for Keystone like Launch an app when inserted, some RGB lights, profile change and pseudo encrypted storage when the key is inserted (actually just a hidden folder)

**Comment by seerge** [comment]
@Valieno hello, I'm sorry but this is extremely exotic thing. I don't know how keystone identifies itself in windows . You can try to do some Autohotkey script to launch some app with that :) 

Anyway, I don't have it and can't add anything.
